geode-protobuf/src/distributedTest/java/org/apache/geode/internal/protocol/protobuf/v1/acceptance/LocatorConnectionDUnitTest.java

```diff
@@ -43,34 +40,54 @@
 import org.apache.geode.internal.protocol.protobuf.v1.ProtobufRequestUtilities;
 import org.apache.geode.internal.protocol.protobuf.v1.serializer.ProtobufProtocolSerializer;
 import org.apache.geode.internal.protocol.protobuf.v1.serializer.exception.InvalidProtocolMessageException;
-import org.apache.geode.test.dunit.DistributedTestUtils;
 import org.apache.geode.test.dunit.Host;
 import org.apache.geode.test.dunit.IgnoredException;
-import org.apache.geode.test.dunit.cache.internal.JUnit4CacheTestCase;
+import org.apache.geode.test.dunit.VM;
+import org.apache.geode.test.dunit.rules.ClusterStartupRule;
 import org.apache.geode.test.dunit.rules.DistributedRestoreSystemProperties;
+import org.apache.geode.test.dunit.rules.MemberVM;
 import org.apache.geode.test.junit.categories.ClientServerTest;
 
 /*
  * Test sending ProtoBuf messages to the locator
  */
 @Category(ClientServerTest.class)
-public class LocatorConnectionDUnitTest extends JUnit4CacheTestCase {
+public class LocatorConnectionDUnitTest {
+
+  // Has to be distributed because how else would we query the members?
+  @Rule
+  public ClusterStartupRule clusterStartupRule = new ClusterStartupRule(2);
 
   @Rule
   public final DistributedRestoreSystemProperties restoreSystemProperties =
       new DistributedRestoreSystemProperties();
 
+  private MemberVM locatorVM;
+  private int locatorPort;
+  private String hostName;
+
   @Before
-  public void setup() throws IOException {
-    Host.getLocator().invoke(() -> System.setProperty("geode.feature-protobuf-protocol", "true"));
+  public void setup() {
+    for (VM vm : VM.getAllVMs()) {
+      vm.invoke(() -> System.setProperty("geode.feature-protobuf-protocol", "true"));
+    }
+
+    Properties properties = new Properties();
+    properties.put(ConfigurationProperties.STATISTIC_SAMPLING_ENABLED, "true");
+    properties.put(ConfigurationProperties.STATISTIC_SAMPLE_RATE, "100");
+
+    clusterStartupRule.getVM(0)
+        .invoke(() -> System.setProperty("geode.feature-protobuf-protocol", "true"));
+    locatorVM = clusterStartupRule.startLocatorVM(0, properties, 0);
+    locatorPort = locatorVM.invoke(() -> ClusterStartupRule.getLocator().getPort());
+
+    clusterStartupRule.startServerVM(1, locatorPort);
 
-    startCacheWithCacheServer();
+    hostName = Host.getHost(0).getHostName();
```

Comment:
Everything in Host is (or is going to be) deprecated. Go ahead and change to:
```
hostName = VM.getHostName();
```

Reply:
oh, nice, thanks! I didn't know that had been added to `VM`.